index.rst

```diff
@@ -107,7 +114,7 @@ Examples
 
 
```

Comment:
Any figure with a particular feature that may deserve to be mentioned?

Reply:
I don't understand what you mean - any figure with regard to the average tracking time being 42 seconds, or do you mean something else?

Reply:
I was thinking that it would be good to have some description of the findings for at least one figure, like guiding the reader.